.eslintrc.js

```diff
@@ -0,0 +1,17 @@
+module.exports = {
+	env: {
+		browser: true,
+		es2020: true
```

Comment:
`es2020` is basically Chrome, Firefox, Edge, Safari, _BUT NOT_ Internet Explorer. It has less around 1% market share, IMO it's not worth taking into account.

See:
- https://kangax.github.io/compat-table/es2016plus/
- https://gs.statcounter.com/browser-market-share